sdk/src/main/java/io/opentelemetry/sdk/internal/TimestampConverter.java

```diff
@@ -46,7 +46,7 @@ public static TimestampConverter now(Clock clock) {
    * @param nanoTime value to convert.
    * @return the {@code Timestamp} representation of the {@code time}.
    */
-  public Timestamp convertNanoTime(long nanoTime) {
+  public Timestamp convertNanoTimeProto(long nanoTime) {
```

Comment:
Probably file an issue to cleanup this TimestampConverter class and remove proto deps.

As discussed today we should discuss a bit before adding more functionality to common.Timestamp if we want to change to use in SpanData and MetricsData a `long epochNanos`

Reply:
#590